test/ui-test/testSuites/suite_messaging/tst_OneToOneChatFlow/test.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Status Desktop One to One Chat Flows
+
+	As a user I want to do basic interactions in a one to one chat.
+
+    The following scenarios cover one to one chat flows with mutual contacts
+
+    The feature start sequence is the following (setup on its own `bdd_hooks`):
+    ** given A first time user lands on the status desktop
+    ** when user logins with username "tester123" and password "TesTEr16843/!@00"
+    ** and the user lands on the signed in app
+
+
+    Scenario: The user can create a one to chat
```

Comment:
Hmm, I don't think `Given` makes sense in this context, no? A `Given` here would be "Given the user is a mutual contact with Athletic", but that's already done in the pre-hook. Here, "user creates a one to one chat with" is more a `When`, because that's what we are testing.

Is the goal to have all scenarios start with a "Given"? Maybe also I'm misunderstanding the BDD meaning of `Given`

Reply:
No, you are not misunderstanding!! It is just only a question of readability. For me it is also ok the `when` here, bc the 3 of the statements are doing the actions, but @glitchminer maybe sees it different (I think, from his other comments in my prs), bc the real action that triggers the validation is `the user sends a chat message xxx`.